packages/cli/src/services/frontend.service.ts

```diff
@@ -359,6 +361,12 @@ export class FrontendService {
 			this.settings.aiCredits.credits = this.license.getAiCredits();
 		}
 
+		Object.assign(this.settings.insights, {
+			enabled: this.modulesConfig.modules.includes('insights'),
+			summary: this.license.isInsightsSummaryEnabled(),
+			dashboard: this.license.isInsightsDashboardEnabled(),
+		});
+
```

Comment:
Some of the other keys in the object only get updated *IF* they're enabled. Should we do the same if the module is enabled? cc @guillaumejacquart

Reply:
I see no reason to apply this condition to update the object :thinking: 
I think you're good to go